packages/diagrams/backend/sirius-components-collaborative-diagrams/src/main/resources/schema/diagram.graphqls

```diff
@@ -36,6 +42,8 @@ type Node {
   targetObjectLabel: String!
   size: Size!
   position: Position!
+  modifiers: [ViewModifier!]!
```

Comment:
Modifiers are never used by the frontend (they are not requested by the GraphQL query)